dev/NavigationView/NavigationView.cpp

```diff
@@ -2671,9 +2671,11 @@ bool NavigationView::BumperNavigation(int offset)
     // that meets the conditions, in the same direction.
     const auto shoulderNavigationEnabledParamValue = ShoulderNavigationEnabled();
     const auto shoulderNavigationForcedDisabled = (shoulderNavigationEnabledParamValue == winrt::NavigationViewShoulderNavigationEnabled::Never);
+    const auto shoulderNavigationOptionalDisabled = (shoulderNavigationEnabledParamValue == winrt::NavigationViewShoulderNavigationEnabled::WhenSelectionFollowsFocus
+        && SelectionFollowsFocus() == winrt::NavigationViewSelectionFollowsFocus::Disabled);
 
     if (!IsTopNavigationView()
-        || !IsNavigationViewListSingleSelectionFollowsFocus()
```

Comment:
Should we remove this function altogether? This is more or less a relic now since we are not using ListView anymore and the function doesn't do anything we couldn't do elsewhere (and we already do in a few cases).

Reply:
It does seem pretty trivial, I think it would probably be easier to parse the code if we removed it

---
In reply to: [471540013](https://github.com/microsoft/microsoft-ui-xaml/pull/3145#discussion_r471540013) [](ancestors = 471540013)